hieradata/node/auxtel-fp01.cp.lsst.org.yaml

```diff
@@ -1,37 +1,85 @@
 ---
-network::interfaces_hash:
-  em2:  # dds-auxtel
-    bootproto: "dhcp"
-    defroute: "no"
-    nozeroconf: "yes"
-    onboot: "yes"
-    type: "Ethernet"
-  em3:  # not connected
-    bootproto: "none"
-    onboot: "no"
-    type: "Ethernet"
-  em4:  # fqdn
-    bootproto: "dhcp"
-    defroute: "yes"
-    onboot: "yes"
-    type: "Ethernet"
-
-network::mroutes_hash:
-  em2:
-    routes:
-      "139.229.147.0/24": "139.229.170.254"
-      "139.229.166.0/24": "139.229.170.254"
-      "139.229.167.0/24": "139.229.170.254"
-      #"139.229.170.0/24": "139.229.170.254"
-      "139.229.178.0/24": "139.229.170.254"
+nm::connections:
+  eno1:
+    content:
+      connection:
+        id: "eno1"
+        uuid: "17b7e246-327c-76fb-6648-b59ca1bdc81e"
+        type: "ethernet"
+        interface-name: "eno1"
+      ethernet: {}
+      ipv4:
+        method: "auto"
+      ipv6:
+        method: "disabled"
+      proxy: {}
+  eno2:  # dds-auxtel
```

Comment:
I'm unsure if CSCs are still running on auxtel-fp01.  I believe the intent was the would only be running on auxtel-mcm.

This seems OK as is but I would ask if the dds interface configuration can be removed.

Reply:
According to Tony they have never used it. I will proceed to clean that up.

https://rubin-obs.slack.com/archives/C07QM70QAR2/p1730829490064959